ambry-clustermap/src/main/java/com/github/ambry/clustermap/HelixClusterManager.java

```diff
@@ -1351,6 +1402,21 @@ private List<ReplicaId> createNewInstance(DataNodeConfig dataNodeConfig, String
       return addedReplicas;
     }
 
+    /**
+     * Delete an instance(node) and remove all disks/replicas associated with it.
+     * @param instanceName the deleted instance
+     * @return a list of replicas present on this instance;
+     */
+    private List<ReplicaId> deleteInstance(String instanceName) {
```

Comment:
this can be folded into deleteInstanceInfo

Reply:
Done